defaults/main.yml

```diff
@@ -11,6 +11,7 @@ os_manila_mount_opts:
   - noatime
   - _netdev # prevents mount blocking early boot before networking available
   - rw
+  - "{{ os_manila_mount_share_info[0].mount_options | default('') }}" # Pass mount opts from manila
```

Comment:
Is it really the desired behaviour that the mount options are only taken from the *first* share and are then used for all? If so, at least I think that needs to be documented.